lib/plugins/aws/package/compile/events/cloudFront/index.js

```diff
@@ -206,11 +210,14 @@ class AwsCompileCloudFrontEvents {
               Object.assign(behavior, event.cloudFront.behavior);
             }
 
+            const { includeBody = false } = event.cloudFront;
```

Comment:
Let's not assign default. Our mechanism should be as:
- If value is not provided (or is `undefined` or `null`) do not set it at all
- Otherwise assign value directly as it is (if someone passed nonsense that validation on AWS side will pick it)